docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+This is the Analytics plan for Firefox Monitor. It documents our use of Google Analytics and what we do with the information we collect.
+
+<br>
+
+## Analysis
+**Firefox Monitor uses Google Analytics to collect and organize data. We do this to get a better understanding of what is working, and where we still have work to do.**
+
+
+**Captured data also helps provide answers to the following questions:**
+
+
+**Demographic:**
+
+>From which country does the majority of our traffic originate?
+
+>In which language does the majority of our traffic view the site?
+
+>Which browsers are most commonly used to access Firefox Monitor?
+
+>Which devices are most commonly used to access Firefox Monitor?
+
+<br>
+  
+**User Behavior:**
+
+>Do users scan their email?
+
+>Do users scan multiple email addresses?
+
+>What percentage of users sign up receive alerts from Firefox Monitor?
+
+>Are users more likely to sign up for alerts before or after scanning their email?
+
+
+<br />
+
+## Collection
+
+Events are reported using the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
+
+We collect data for the following events:
+
+- When the page loads (or reloads).
+`hitType` : pageView
+`page` : location.pathName
+<br>
+
+#### Forms:<br><br> `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form` &nbsp;
+
+- When a form appears on the page.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : View
+`eventLabel` : Page location ID.
+
+
+- When a user focuses inside a form input.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Engage
+`eventLabel` : Page location ID.
+
+
+- When a user submits a form.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Submit
+`eventLabel` : Page location ID.
+
+
+- When a user submits an invalid email.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Failure
+`eventLabel` : Page location ID.
+
+
+- When a user submits a valid email.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Success
+`eventLabel` : Page location ID.
+<br>
+
+
+#### Buttons:<br><br> `Download Firefox Button`&nbsp;&nbsp;`Sign Up Button`&nbsp;&nbsp; `See Full Report Button`<br><br> `Sign up to Check Button` &nbsp;`Show All Breaches Button`
+
+
+- When a button appears on the page
+`hitType` : event
+`eventCategory` : Button ID
+`eventAction` : View
+`eventLabel` : Page location ID.
+
+
+- When a user clicks a link or button.
+`hitType` : event
+`eventCategory` : Button or Link ID
+`eventAction` : Engage
+`eventLabel` : Page location ID.
+
+
+<br>
+
+#### Links:<br><br>`Outbound & Utility Links` &nbsp;&nbsp; `Internal Links`
+  
+
+- When a user clicks a link
+`hitType` : event
+`eventCategory` : Type of Link (Outbound or Internal)
+`eventAction` : Click
+`eventLabel` : Link ID // Page location ID.
+<br>
+
+## Opt Out of Google Analytics Tracking
+**Firefox Monitor respects user privacy and honors **DNT** headers.**
```

Comment:
Note that the `**DNT**` bolding doesn't really _do_ anything here since the entire sentence is already bold. But looking at GitHub preview, it doesn't actively hurt anything either.

Not sure if you want to convert to `_DNT_` for italics for emphasis in the bold sentence, or leave as-is.

**Firefox Monitor respects user privacy and honors _DNT_ headers.**